modules/support_ticket/config/optional/migrate.migration.d6_support_ticket.yml

```diff
@@ -0,0 +1,64 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - migrate.migration.d6_filter_format
+    - migrate.migration.d6_node_settings
+    - migrate.migration.d6_node_type
+    - migrate.migration.d6_user
+  module:
+    - support_ticket
+id: d6_support_ticket
+migration_tags:
+  - 'Drupal 6'
+label: 'Drupal 6 support tickets'
+load: {  }
```

Comment:
Adding links for reference: 
d.o issue: https://www.drupal.org/node/2549013
change record: https://www.drupal.org/node/2558851

This configuration was created with a core checkout from 4 days ago, before the above patch landed. Is it enough to just remove line 15? (And the same line from the other two migrations)